qemu/tests/numa_memdev_options.py

```diff
@@ -0,0 +1,91 @@
+import logging
+
+from avocado.utils import process
+
+from virttest import error_context
+from virttest import utils_misc
+
+
+def check_host_numa_node_amount(test, params):
+    """
+    Check host NUMA node amount
+    """
+    host_numa_nodes = int(process.system_output("%s" % params['numa_hardware_cmd']).split()[1])
+    if host_numa_nodes < 2:
+        test.cancel("The host numa nodes should be at least 2! But there is %d." % host_numa_nodes)
+
+
+def check_query_memdev(test, params, vm):
+    """
+    Check memory info in query-memdev
+    """
+    mem_devs = params['mem_devs'].split()
+    query_list = vm.monitor.info("memdev")
+    if len(mem_devs) != len(query_list):
+        test.fail("%d memory devices in query-memdev, but not %d!"
+                  % (len(query_list), len(mem_devs)))
+    policy = params['policy_mem']
+    for dev in query_list:
+        mem_dev = dev['id'].split('-')[1]
+        if dev['policy'] != policy:
+            test.fail("memdev = %s: 'policy' is '%s', but not '%s'!"
+                      % (mem_dev, dev['policy'], policy))
+        prealloc = (params['prealloc_%s' % mem_dev] == 'yes')
+        if dev['prealloc'] != prealloc:
+            test.fail("memdev = %s: 'prealloc' is not '%s'!"
+                      % (mem_dev, params['prealloc_' % mem_dev]))
+        if policy == 'default':
+            continue
+        host_node = str(dev['host-nodes'][0])
```

Comment:
better use "if ...esle ...."

Reply:
There is no difference in functionality, but your choice is more clear for readers. I'll modify it. Thanks!